test/components/tsconfig.json

```diff
@@ -12,7 +12,6 @@
     "resolveJsonModule": true,
     "moduleResolution": "Bundler",
     "useDefineForClassFields": true,
-    "composite": true,
```

Comment:
Why delete this

Reply:
I check the Typescript document，maybe no need composite:true for app project?

https://cspotcode.com/posts/tsconfig-composite-configuration

https://www.typescriptlang.org/docs/handbook/project-references.html

Reply:
`components` is a lib project, not app

Reply:
ok, fixed this one.